### PR TITLE
Uamqp updates

### PR DESCRIPTION
**Pull request by yunhaoling**
- [x] Remove executor
- [x] Call private encode message
- [x] Timeout calculation in milliseconds

### CODE REVIEW

File: uamqp/message.py [hunk 1]
```diff
@@ -596,7 +596,7 @@ def gather(self):
             try:
                 if not data.application_properties and self.application_properties:  # Message-like object
                     data.application_properties = self.application_properties
-                message_bytes = data.encode_message()
+                message_bytes = data._encode_message()  # pylint: disable=protected-access
```
Comment (annatisch) [inline uamqp/message.py]: We need to support both `encode_message` and `_encode_message`

Comment (yunhaoling) [reply uamqp/message.py]: you're right, otherwise we lose compatibility.

**Comment by annatisch** [comment]
Do you want to add the version bump/release notes etc to this PR or separately?

**Comment by yunhaoling** [comment]
@annatisch  we can do this separately

**Pull request merged**